services/distribution/src/main/resources/main-config/v2/plausible-deniability-parameters.yaml

```diff
@@ -0,0 +1,5 @@
+#plausible-deniability-parameters:
+  # validation rules for checkInSizesInBytes:
+  #   * zero entries are allowed
+  #   * each entry must be >= 0
+  check-in-sizes-in-bytes: [10, 20]
```

Comment:
Can this be made more strict?

Reply:
same, as above. We can sync with @mlenkeit